resourcecontrollerv2/resource_controller_v2_integration_test.go

```diff
@@ -1007,6 +1008,7 @@ var _ = Describe("Resource Controller - Integration Tests", func() {
 			fmt.Fprintf(GinkgoWriter, "GetResourceKey() result:\n%s\n", common.ToJSON(result))
 
 			Expect(*result.ID).To(Equal(testInstanceKeyCRN))
+			Expect(result.OnetimeCredentials).ToNot(BeNil())
```

Comment:
`result.OnetimeCredentials` is a pointer so please dereference it before checking its value.
```suggestion
			Expect(*result.OnetimeCredentials).ToNot(BeNil())
```

Reply:
Addressed. Please verify